inference-engine/tests/functional/inference_engine/serialization/single_layer/group_convolution_backprop_data.cpp

```diff
@@ -26,22 +26,25 @@ const std::vector<std::vector<size_t>> kernels = {{3, 3}};
 const std::vector<std::vector<size_t>> strides = {{1, 1}};
 const std::vector<std::vector<ptrdiff_t>> padBegins = {{0, 0}};
 const std::vector<std::vector<ptrdiff_t>> padEnds = {{0, 0}};
+const std::vector<std::vector<ptrdiff_t>> outputPadding = {{}, {1, 1}};
 const std::vector<std::vector<size_t>> dilations = {{1, 1}};
 const std::vector<size_t> numOutChannels = {8, 16};
 const std::vector<size_t> numGroups = {2, 8};
 const std::vector<ngraph::op::PadType> pad_types = {
     ngraph::op::PadType::EXPLICIT, ngraph::op::PadType::VALID,
     ngraph::op::PadType::SAME_LOWER, ngraph::op::PadType::SAME_UPPER};
 const auto inputShapes = std::vector<size_t>({1, 16, 30, 30});
+const std::vector<std::vector<size_t >> emptyOutputShape = {{}};
 
 const auto groupConvBackpropData2DParams = ::testing::Combine(
     ::testing::ValuesIn(kernels), ::testing::ValuesIn(strides),
     ::testing::ValuesIn(padBegins), ::testing::ValuesIn(padEnds),
     ::testing::ValuesIn(dilations), ::testing::ValuesIn(numOutChannels),
-    ::testing::ValuesIn(numGroups), ::testing::ValuesIn(pad_types));
+    ::testing::ValuesIn(numGroups), ::testing::ValuesIn(pad_types),
+    ::testing::ValuesIn(outputPadding));
 
-INSTANTIATE_TEST_SUITE_P(
-    smoke_GroupConvBackpropData2D_Serialization, GroupConvBackpropDataLayerTest,
+INSTANTIATE_TEST_CASE_P(
```

Comment:
Please rebase to the latest master and use `INSTANTIATE_TEST_SUITE_P` macro.

Reply:
Done.